sulfur_simulation/show_simulation.py

```diff
@@ -77,3 +157,42 @@ def colorize(val: bool) -> str:  # noqa: FBT001
         lines.append(" ".join(colored_row))
 
     return "\n".join(lines)
+
+
+def create_jump_plot(jump_counter: np.ndarray, sampled_jumps: np.ndarray) -> Figure:
+    """Plot attempted and successful jump counts."""
+    delta = np.array(
+        [
+            (-1, -1),
+            (-1, 0),
+            (-1, 1),
+            (0, -1),
+            (0, 0),
```

Comment:
this should be a global constant - this is the third time it has been defined, what if they were defined diffreently?